src/framework/uicomponents/view/popupview.cpp

```diff
@@ -547,13 +534,21 @@ bool PopupView::alwaysOnTop() const
     return m_alwaysOnTop;
 }
 
+void PopupView::setupAlwaysOnTop()
+{
+    if (isDialog()) {
+        m_window->setAlwaysOnTop(m_alwaysOnTop);

```

Comment:
Here, you need to check if `m_window` is not nullptr. Otherwise, this causes a crash whenever running a QML plugin, because of `src/plugins/view/pluginview.cpp:183`